kernel/BT.c

```diff
@@ -738,10 +738,41 @@ void BTUpdateRegisters(void)
 	if(inited == 0)
 		return;
 
-	if(intr == 1)
-	{
-		intr = 0;
-		__readintrdataCB();
+   // Try to load Wiimote/Classic Controller bindings from ircc.ini
+    const char *filenames[2] = {
+		file_sd, file_usb,
+        "sd:/ircc.ini",

```

Comment:
Why not in the app directory? 
Cluttering up the root directory with all kinds of files is not nice.